facia-tool/app/tools/FaciaToolArchive.scala

```diff
@@ -1,53 +0,0 @@
-package tools
-
-import awswrappers.dynamodb._
-import com.amazonaws.regions.{Region, Regions}
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
-import com.amazonaws.services.dynamodbv2.model.{AttributeValue, PutItemRequest}
-import common.{ExecutionContexts, Logging}
-import conf.{Switches, Configuration}
-import implicits.JsonImplicits._
```

Comment:
This was the last use of `JsonImplicits` (introduced January 2015 with https://github.com/guardian/frontend/pull/8059) in the Frontend codebase, so that could also have been deleted too!

We found the obsolete file while doing https://github.com/guardian/frontend/pull/25190 in July 2022, and opened PR #25331 to remove it!

Reply:
👋 so long little guy